netlify.toml

```diff
@@ -1,7 +1,7 @@
 [build]
-  base = "book/website"
-  command = "pip install -r requirements.txt && jupyter-book build ."
-  publish = "_build/html"
+base = "book/"
+command = "python -m pip install --upgrade pip && pip install -r website/requirements.txt && python -m pathways.main pathways website/"
+publish = "website_copy/_build/html"
```

Comment:
Why does this change?

Reply:
This is because the feature applied here for pathway creation is through a package we developed last year. Arya has been working over the last few months to improve its usability as deployed here: https://deploy-preview-3277--the-turing-way.netlify.app/#different-pathways.

Reply:
Sorry, I should've been clearer: why `website_copy/_build` rather than `_build`

Reply:
Good point! I will let @arya1302 and @likeajumprope address that. Thank you for pointing to that.

Reply:
As part of the pathway feature implementation, the book is built into the 'website_copy' directory (please check lines 59, 60, and 87 of[ https://github.com/alan-turing-institute/bio-Turing-Way/blob/main/pathways/pathways/main.py](https://github.com/alan-turing-institute/bio-Turing-Way/blob/main/pathways/pathways/main.py)). The site contents will only be found in the 'website_copy/_build/html' directory. So this edit was necessary for deploying the site to Netlify.

Reply:
But if the intention is to make this the default way of deploying the book, why not remove the "new_path" stuff in the Python file, and continue to use the current directory?

Reply:
> Arya is testing the best way to deploy the package

_from https://github.com/the-turing-way/the-turing-way/pull/3277#issuecomment-1722568110_

Maybe we are still in the testing phase

Reply:
> But if the intention is to make this the default way of deploying the book, why not remove the "new_path" stuff in the Python file, and continue to use the current directory?

This was implemented in 2022 when the Python package was developed. I'm not entirely sure about the specific reasons behind it. If necessary, I can certainly try removing the 'new_path' and use the current directory instead.

Reply:
maybe we could already make the pathways deployment the "default" way but keep the old way with "old_path"..

Reply:
I will try removing the website_copy directory and see if everything works fine.